### ISSUE TITLE
Extract stream crashes when no files are in the input stream

### ISSUE THREAD

**Issue by mattbrunetti** [gabegorelick/gulp-angular-gettext#39]
If I pipe a vinyl stream that has no files into a gulp-angular-gettext `extract` stream, I get the following error:

```
C:\Users\mbrunetti\Documents\mtdirect-next\packages\mtdirect-ui\node_modules\gulp-angular-gettext\lib\extract.js:74
      cwd: firstFile.cwd,
                    ^
TypeError: Cannot read property 'cwd' of null
    at DestroyableTransform.flush [as _flush] (C:\Users\mbrunetti\Documents\mtdirect-next\packages\mtdirect-ui\node_modules\gulp-angular-gettext\lib\extract.js:74:21)
    .....
```

It seems like the `flush` function expects that `firstFile` has been set, although it may not have been.

**Comment by stenmuchow** [comment]
+1

**Comment by gabegorelick** [comment]
This should be fixed in v2.3.0. Let me know if you have any issues.
